### PR TITLE
67 docs for template provider

### PR DESCRIPTION
**Pull request by cloudguruab**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update. 

- **What is the current behavior?** (You can also link to an open issue here)
Issue #67 documentation for create provider

- **What is the new behavior (if this is a feature change)?**
n/a 

- **Other information**:
Provided updates to help communicate initialization, description and code snippets for end users.

### CODE REVIEW

File: docs/docs/community/create-provider.md [hunk 1]
```diff
@@ -7,6 +7,91 @@ All of our providers need to implement one or more of our provider interfaces, b
 
 For a provider template you can copy one of our existing provider in the `providers` folder in the notifire project, make the relevant changes and create a PR against the monorepo.
 
+## Description
+
+Providers allow us to handle message delivery over multiple channels. We have multiple providers for each channel (SMS, Email, Push, Direct, and orthers). To get started with adding a new provider lets look at setting up our repository.
```
Comment (scopsy) [inline docs/docs/community/create-provider.md:12]:
```suggestion
Providers allow us to handle message delivery over multiple channels. We have multiple providers for each channel (SMS, Email, Push, Direct, and others). To get started with adding a new provider let's look at setting up our repository. 
```

File: docs/docs/community/create-provider.md [hunk 2]
```diff
@@ -7,6 +7,91 @@ All of our providers need to implement one or more of our provider interfaces, b
...
+
+```zsh
+yarn run v1.22.17
+$ npx hygen provider new
+? What type of provider is this? … 
+❯ EMAIL
+  SMS
+```
+
+For this example we will be selecting `EMAIL` as our provider type. The name for our provider will be exampleProvider.
```
Comment (scopsy) [inline docs/docs/community/create-provider.md:46]:
```suggestion
For this example, we will be selecting `EMAIL` as our provider type. The name for our provider will be exampleProvider.
```

File: docs/docs/community/create-provider.md [hunk 3]
```diff
@@ -7,6 +7,91 @@ All of our providers need to implement one or more of our provider interfaces, b
...
+
+```zsh
+? Write the provider name camelCased: › exampleProvider
+```
+
+Once our exampleProvider is generated we will need to begin working from within `/providers/exampleProvider` to begin adding our provider. Be sure to write test alongside your provider. See below for template examples for our `exampleProvider`.
```
Comment (scopsy) [inline docs/docs/community/create-provider.md:52]:
```suggestion
Once our exampleProvider is generated we will need to begin working from within `/providers/exampleProvider` to begin adding our provider. Be sure to write the test alongside your provider. See below for template examples for our `exampleProvider`.
```

**Review by scopsy — COMMENTED** [review]
This looks amazing! Just a few small suggestion and a little typo and we can merge this 🎉🎉🎉

**Comment by cloudguruab** [comment]
Awesome, I will go ahead and make some adjustments soon! Thanks again for the help.

**Pull request merged**